stdlib/public/core/SmallString.swift

```diff
@@ -295,23 +312,27 @@ extension _SmallString {
 #endif
 
 extension UInt64 {
-  // Fetches the `i`th byte, from least-significant to most-significant
-  //
-  // TODO: endianess awareness day
+  // Fetches the `i`th byte, from left to right.
   @inlinable @inline(__always)
```

Comment:
What does "left to right" mean, memory-order/host-endianess?

Reply:
Clarified the comment (memory-order).